src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -599,4 +605,40 @@ private String getElementSrcWithRetryAfterWaitForPresence(By by) {
         }
     }
 
+    /**
+     * Expands a particular panel, causing its results to load.
+     * @param panelNumber the position of panel element
+     */
+    public void loadResultPanel(int panelNumber) {
```

Comment:
If this is only used for question panels, a better method name would be `loadResultQuestionPanel`. The parameter name can be `questionNumber`.